frontend/src/app/admin/Admin.tsx

```diff
@@ -32,6 +32,10 @@ const Admin = () => {
                   Create new device type
                 </Link>
               </div>
+              <div>
+                {" "}
```

Comment:
**question (non-blocking):** What's up with this whitespace?

Reply:
I copied it from the entries above... there doesn't seem to be any downside to removing all of them.